sdk/Trace/TraceContext.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\Sdk\Trace;
+
+use OpenTelemetry\Trace as API;
+
+final class TraceContext implements API\TextMapFormatPropagator
+{
+    public const TRACEPARENT = 'http_traceparent';
+    public const TRACESTATE = 'http_tracestate';
+
+    // TODO Consolidate these and the validity checks in SpanContext.php
```

Comment:
This just looks to be a superset of the validity checks in [SpanContext.php](https://github.com/open-telemetry/opentelemetry-php/blob/master/sdk/Trace/SpanContext.php) for reference.

Reply:
Yes, you're right. The validity checks here are the same in `SpanContext.php`. I'll add some clean up here in a follow-up PR to consolidate those checks.